### ISSUE TITLE
Tests for goodness of fit

### ISSUE THREAD

**Issue by Ali-Rey** [dataanalysis3/datan3_2019#20]
I have conducted tests (r squared, residuals) to test the accuracy of some of my graphs. I have commented on their results in my analysis however did not include the output (The tables and residual graphs) in the final document as it would go way over the output limit. Should I include them, submit a separate file with the code for these results, or leave them be?

**Comment by abessudnov** [comment]
I don't think this is essential (the report is not so much about statistical modelling) so I don't think you should include this.
